.github/workflows/_unit-tests-python.yml

```diff
@@ -59,5 +59,5 @@ jobs:
         with:
           working-directory: ${{ inputs.working-directory }}
           files: ./coverage.xml
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
Yes, I used a separate PR for that: https://github.com/huggingface/datasets-server/pull/720#issuecomment-1408447822, and merged it here...
Maybe it would have been better to send to main... Well, next time